charts/github-inactivity/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Default values for github-inactivity.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  # -- container repository, adjust in https://github.com/saritasa-nest/saritasa-devops-docker-images/pull/29
+  repository: public.ecr.aws/saritasa/github-inactivity
+  # -- pull policy
+  pullPolicy: IfNotPresent
+  # -- Overrides the image tag whose default is the chart appVersion.
+  tag: "0.1"
+
+# -- name of the environment you're placing the github-inactivity for like dev, prod, staging
+environment: ""
+
+# -- not allow concurrent job builds
+concurrencyPolicy: "Forbid"
+
+# -- run job in the 1st day of the month
+schedule: "0 0 1 * *"
+
+successfulJobsHistoryLimit: 5
+failedJobsHistoryLimit: 5
+
+# -- credentials for docker login
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: true
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use. If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+cronJobSecurityContext: {}
```

Comment:
make default to be strict
```yaml
 securityContext:
            allowPrivilegeEscalation: false
            privileged: false
            runAsNonRoot: true
            runAsUser: 999
            runAsGroup: 999
            capabilities:
              drop:
                - all
```

Reply:
done in this PR https://github.com/saritasa-nest/saritasa-devops-github-inactivity/pulls